snapcraft/internal/lifecycle/_init.py

```diff
@@ -23,6 +23,7 @@
 _TEMPLATE_YAML = dedent(
     """\
     name: my-snap-name # you probably want to 'snapcraft register <name>'
+    base: core18
```

Comment:
This looks naked without an informational comment like everything else. Maybe something as simple as `# the base snap is the execution environment for this snap`?